src/apollo-server/index.ts

```diff
@@ -159,6 +166,11 @@ export class OttofellerApolloServerProject extends TypeScriptAppProject implemen
     PullRequestTest.addToProject(this, {...options, isLighthouseEnabled: false})
     ProjenDriftCheckWorkflow.addToProject(this, options)
 
+    // ANCHOR Code owners
+    if (this.github && options.codeOwners) {
+      new CodeOwners(this.github, {codeOwners: options.codeOwners})
```

Comment:
@quesabe with this codeowners input param do we make the CODEOWNERS file managed by projen?

Reply:
Yes, the file is managed by `projen`. It can be set as `readonly`, but all manual changes would be overwritten after `npx projen`. Thus only the rows provided via `options.codeOwners` are persistent.